### DISCUSSION TITLE
Setting up alerts

### DISCUSSION THREAD

**Discussion by TomYoungs** [polymind-inc/acmebot#482] [Q&A] [answered]
Hey love the project,

just wondered about logging/alerts, is there a way to alert the users if a certificate fails to renew etc? Ideally it would be over email since that's what we use for our other alerts

Thanks in advance, Tom

**Comment by shibayan** [comment, marked as answer]
Supports notification via Webhook of successful or failed certificate renewal. I recommend setting the Incomint Webhook URL for Slack and Teams as it will send properly formatted messages.

https://github.com/shibayan/keyvault-acmebot/wiki/Getting-Started#2-add-application-settings

Generic webhook URLs can also be specified, but if email notifications are required, this will need to be accomplished using a combination of Logic Apps or similar.
